File: java/Motherboard.java
public class Motherboard extends Komponen {
    private String chipset;
    private int slotRam;

    public Motherboard(String merk, String nama, String chipset, int slotRam) {
        super(merk, nama);
        this.chipset = chipset;
        this.slotRam = slotRam;
    }

    public String getChipset() {
        return chipset;
    }

    public int getSlotRam() {
        return slotRam;
    }

    public void setChipset(String chipset) {
        this.chipset = chipset;
    }

    public void setSlotRam(int slotRam) {
        this.slotRam = slotRam;
    }

    @Override
    public String toString() {
        return super.toString() + " | Chipset: " + chipset + " | Slot RAM: " + slotRam;
    }
}

File: py/motherboard.py
from komponen import Komponen

class Motherboard(Komponen):
    def __init__(self, merk: str, nama: str, chipset: str, slot_ram: int):
        super().__init__(merk, nama)
        self.chipset = chipset
        self.slot_ram = slot_ram

    def get_chipset(self):
        return self.chipset

    def get_slot_ram(self):
        return self.slot_ram
